source/default/_patterns/00-protons/legacy/css/components/UTC-custom-blocks/_utc_hover_images.css

```diff
@@ -0,0 +1,133 @@
+.utc-image-hover { display:flex!important }
+
+figure.utc-hover-image-effect * {
+  @apply box-border;
+}
+figure.utc-hover-image-effect {
+  box-shadow: 5px 5px 10px rgb(0 0 0 / 0.5);
+}
+
+figure.utc-hover-image-effect img {
+  @apply max-w-full opacity-100 w-full;
+  -webkit-transition: opacity 0.35s;
```

Comment:
that's interesting, I rarely see webkit calls on vanilla css. Was this compiled from less or SASS into css? I think postcss can add the vendor prefixes if we used the native css class `transition: opacity .25s`.

Reply:
Ok. Old school habit, I guess.